Indicators/HeikinAshi.cs

```diff
@@ -112,6 +117,16 @@ protected override decimal ComputeNextValue(IBaseDataBar input)
                 Low.Update(input.Time, Math.Min(input.Low, Math.Min(Open, Close)));
             }
             
+            // IBaseDataBar subtypes TradeBar and RenkoBar have a Volume property, but QuoteBar does not.
+            var volume = 0.0m;
+            if (input is TradeBar)
+                volume = ((TradeBar) input).Volume;
+            else if (input is RenkoBar)
+                volume = ((RenkoBar) input).Volume;
```

Comment:
Nice fix! 👍 
nit - in Lean the coding style is usually to add all brackets, so for this case would be:
```
if (input is TradeBar)
{
    volume = ((TradeBar) input).Volume;
}
else if (input is RenkoBar)
{
    volume = ((RenkoBar) input).Volume;
}

```

Reply:
Thanks! I 100% support nit-picking for consistent syntax! Nothing makes me twitch more when reading code, haha. 😄 I'll revise. 🙂